### PR TITLE
refactor:  changed conditional styling

### PR DESCRIPTION
**Pull request by LilianSemann**

### CODE REVIEW

File: src/app/page.tsx
Comment (Narue-GS) [inline src/app/page.tsx:1]: código puro na page, evite isso. Coloque o conteúdo dentro do componente

File: src/app/components/list.tsx
Comment (Narue-GS) [inline src/app/components/list.tsx:1]: the name of your interface is Types. This is not a good practice, the name of your types should be something useful

Comment (Narue-GS) [inline src/app/components/list.tsx:1]: You're importing an interface called Types from a file called types.tsx in a folder called types. Just don't.

Comment (Narue-GS) [inline src/app/components/list.tsx:1]: You're importing an interface called Types from a file called types.tsx in a folder called types. Just don't.

File: src/app/page.tsx
Comment (LilianSemann) [reply src/app/page.tsx:1]: thanks man for the help :D

**Review by Narue-GS — APPROVED** [review]
Nice code, smart solutions. Aproved!

**Pull request merged**